core/src/main/java/org/acegisecurity/providers/ldap/authenticator/BindAuthenticator2.java

```diff
@@ -1,62 +0,0 @@
-/*
- * The MIT License
- * 
- * Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
- * 
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- * 
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- * 
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package org.acegisecurity.providers.ldap.authenticator;
-
-import org.acegisecurity.ldap.InitialDirContextFactory;
-import org.acegisecurity.userdetails.ldap.LdapUserDetails;
-
-import java.util.logging.Logger;
-import java.util.logging.Level;
-
-/**
- * {@link BindAuthenticator} with improved diagnostics.
- * 
- * @author Kohsuke Kawaguchi
- */
-public class BindAuthenticator2 extends BindAuthenticator {
```

Comment:
Seems to have been unused, and I do not want to depend on Spring Security modules for LDAP here; belongs in the `ldap` plugin only.

Reply:
Correction: _was_ used by the `ldap` plugin, and should have been there all along (see 9b4cd99 & 32cba8f). Correcting in https://github.com/jenkinsci/ldap-plugin/pull/46/commits/a5267e6c0125984971c885461903e824b8c8b34b.